### ISSUE TITLE
Underpowering discussion

### ISSUE THREAD

**Issue by JohN100x1** [JohN100x1/IsekaiMod#159]
According to users, the class has become too powerful and need to be reworked.
> [17 April 2023, 5:23PM](https://forum.nexusmods.com/index.php?showtopic=12204503/#entry123322578)
I absolutely love this mod. I really like the growth of a lot of the abilities and just how far outside the power curve it pushes everything. However I find some of the abilities are a little too much and it can be hard, particularly with the Overpowered Abilities, to limit them enough to not kill everything. I have a few suggestions, if I may:
>
> 1. For Overpowered Abilities: Add "Power Limiter" - 'You aren't ready for the true awesomeness of your power - yet.' (Adds a 20th level ability that grants things like Supreme Being, Unlimited Power, Super Buff, Super Spell Penetration (+20), or the combo of Empowered/Maximized/Intensified Spell all at once but requires several ranks in Power Limiter to select.)
>
> 2. Again for Overpowered abilities, I feel like there needs to be simpler selections that aren't quite so overpowered. For example: +1 damage per die rolled of all spells but you can take it multiple times, +2 to dodge AC, or a 'nothing' stat like +1 to your harem.
>
> 3. I'd also really like to see a "Sidekick" class (perhaps a prestige class) that I could give to companions. Basically it would still give a lot of the same bonuses that the other Isekai classes give but remove or significantly lessen some of the initial bonuses like "Plot Armor" and remove spell levels.

> [26 April 2023, 7:49PM](https://forum.nexusmods.com/index.php?showtopic=12204503/#entry123588690)
6.0.0suggestions -
I want to preface this post by saying that most of the additions in this mod are fantastic and very in line with the theme and the mod’s goals.
These suggestions are intended to keep the theme of the Isekai Protagonist Class and its Archetypes while making some mechanical
changes to prevent lots of features from turning the class into an "I Win class". There's nothing wrong with including "I Win" abilities either.
>
> Edge Lord Archetype – Drop or lower the spells from the Edge Lord, make it more of a pure fighter class. Extra spells just get in the way.
>
> Plot Armor – Luck bonus to AC equal to half your class level. Drop the bonus to saves. Other abilities grant bonuses to saves and it becomes
a mechanical issue of always passing if you keep the bonus too.
>
> Gifted –Just give the competence bonus. Again, this is a mechanical issue where getting both is a frequent auto-pass. You also have
Meta-Luck as an Overpowered Ability which does the same thing.
>
> Nascent Apotheosis – Drop the attribute bonus and spell resistance. You have other abilities that grant these. Since they don’t stack,
you’re just eliminating some other choices. Add immunity to Level Drain.
>
> Signature Strike – This ability deals considerably more damage than the other two even if it requires flanking. Since the game
simplifies flanking to “Two creatures attacking the same target” this becomes unreasonably more damaging. Modify to 3d6 extra
or double DEX bonus damage. This keeps it in line with the other Signature abilities.
>
> Energy Barrier – Should be a flat +4 to AC and I would choose a Dodge bonus since they stack. A natural armor enhancement bonus would
also be reasonable. Note that just a natural armor bonus wouldn’t stack, it needs to be an enhancement. See the Barkskin spell
>
> Extra Strike (Edge Lord) – Don’t apply this to the off-hand.One extra attack is already fantastic.
>
> Siphoning Aura – This should probably be a level drain effect like Enervation or Energy Drain.
>
> Majestic Aura – A +4 bonus to all attributes (as if everyone is under the effect of all the 2nd level buff spells) is also plenty. I also might
suggest making it a selection similar to the Resist Energy spells. You can choose which 2nd level buff to permanently apply to everyone.
>
> Hax (Serious Strike) – Fort Save DC40 or take 20d6 extra damage (each hit). This is still tons of extra damage
>
> Hax (Invincibility) – DR 30/Mythic (only mythic attacks bypass, and that’s still a lot of DR), add immunity to level drain. DR 30 is
enough to almost entirely negate attacks from some of the most powerful creatures in Table Top.
Using DR /Mythic (epic in TT) means that non mythic creatures are a non-threat to you.
>
> Hax (Faster than Light) – Permanent Haste, Auto-Quicken spells, +10 dodge AC, +100 initiative

> [26 April 2023, 7:44PM](https://forum.nexusmods.com/index.php?showtopic=12204503/#entry123588576)
6.0.0 suggestions (cont'd)-
>
>Overpowered Ability changes:
>
> Dupe Gold – change to 10,000 or perhaps just get 1M gold. Wealth by Level says that a 20th level character should have 880k gold in equipment and coin.
>
> Instakill – Change to DC20 + Class Level  (The save for Power Word Kill is DC19 +Ability Bonus.) Creatures would still be affected frequently, but not 95% of the time.  
>
> Mind Control – Limit this to one creature at a time if it isn’t already.
>
> Power Leveling – Drop the 120ft radius (this is probably slowing the game down since it would be checking the radius all the time.)
>
> Super Buff – There are a lot of overlapping abilities here.This should probably just extend the duration of any given buff to 24 hours.  
Otherwise narrow the list down. (For example, Resist Energy and Protection From Energy don't stack.)
>
> Supreme Being – Consider a prerequisite of class level 15. This one is a personal opinion, but unless your name is Saitama, you probably don't achieve this until the last season.
>
> Overpowered Aura – Change the alignments to only any good, any evil, any chaotic, and any lawful. This lets someone take two versions. Make
the damage the same across these; they are different right now. 5d6 extra is fine, even if it’s a little high (avg. of 16.5 dmg). Personally, I’d assume
that someone is going to take two of these and lower the damage to 2d8 each (avg. of 9).

> [26 April 2023, 7:38PM](https://forum.nexusmods.com/index.php?showtopic=12204503/#entry123588447)
6.0.0 suggestions (cont'd)-
>
> New Overpowered abilities:
>
> Prescient: +20 to initiative
Ward: +4 to all saves
Greater Ward: +4 to all saves, stacks with bonus from Ward
Cheat code: Gives you several end game items like Celestial Armor, a Holy Avenger sword, +5 armor, +5 amulet of mighty/agile fists, a Robe
of the Archmage, etc.
Werewolf of Passwall Street: This one is difficult, but I’ll cite ToyBox for it being possible. This ability modifies vendor buy prices to
be 500% of value.

**Comment by Necht** [comment]
That sounds interesting. The Hax ability definitely needs to be weakened, because it is too broken right now. 

As for the rest of the abilities, I'm not sure that they need to be changed much. Some, perhaps, but not all.

Plot Armor and Dupe Gold definitely do not need to be changed.

But if you weaken all this, then the God-Emperor will look too broken against the background of other archetypes. He already looks much stronger than all the others. 
And in this case, it also needs to be weakened. Instead of being immune to damage, let there be extremely high resistance to physical and energy damage. After all, the God-Emperor can be wounded and killed in Warhammer 40k, it's just too difficult.

**Comment by Mythalar** [comment]
Hax is indeed too strong, and I would dare say not the most interesting since it makes some previous bonuses/abilities useless.
Plot armor could be half level, still very powerful as a luck bonus (but would let the save bonus in).
Quite agree on Gifted also since it'a partly redundant with Meta-luck (competence bonus is fine).

But I don't agree with most of the other points.
Or actually I don't agree with the end : yes, if you choose all the broken abilities together you blow away anything but you don't have to.

In the end it is an RPG and, for one, I usually choose Auto-Meta - Reach for QoL and it does not double my damages.
There are vanilla classes/builds that can ruin your game if you always use the same busted mechanic...

Some abilities, while strong, can also be seen as QoL : Majestic aura gives a rare sacred stat bonus yes, but in one of my runs it allowed me to not have all buffs up at every fight.

Regarding God-Emperor (yes it's my fav, but the base IP is actually just after!), it is indeed strong but not that much more than other archetypes since quite everything is level-scaled and you 3/4 of the game is not at lvl 20. I would argue that the Overlord is quite more brutal with OP abilities (yet again, you don't have to take ALL the more powerful ones).
Yet I agree with @Necht that he does not need to be immune to all damage : the full immunity part of the captsone could be changed to regeneration/fast healing and/or a quite steep missing chance (50%?) since lesser beings are complelled to bow before you (the lvl17 energy immunity is fine since it can be bypassed by foes).

**Comment by Mythalar** [comment]
Readily agree with @NightOwlByDay, especially on this : 

>Not having to take those other abilities frees those "ability slots" up to take other abilities, meaning more choices and more customizability. Or one can choose to take those other abilities as well and indeed achieve Saves that most enemies can't beat, also an option. Removing the bonus to Saves also removes / limits those optins for character building.

You also are right with difficulty settings with Toybox, I personaly use the enemy HP multiplier so I can actually use all my sweet spells and abilities !

**Comment by NightOwlByDay** [comment]
I just realised that my last post on this topic consisted mostly on what not to do / change, but had nothing in the form of offering alternatives / my on suggestions on how to address the overpowered nature of the Isekai Protagonist class. Which does not make for constructive feedback.

Having already stated my preference for role playing and not limiting the current variety of possible builds, my suggestions would be as follows:

Introduce an equivalent to the new “Underpowered Abilities” to the special powers' selection. Which would be “Normal Powers” I suppose?
A few ideas:

- Health and Safety Training: You attended a course on best practices to keep yourself hale and healthy during your adventuring days in this new world you found yourself in. Don't go lifting heavy things with your back!
Effect: +10 to max Health

- Fork Lift Certified: You were forklift certified in your old life and know how to move a lot of weight around. Now, to find a forklift around here.
Effect: + 300 to Carry Capacity

- Spice Enthusiast: You love spicy food! The spicier, the better.  Eating all that spice gave you a pretty strong stomach.
Effect: +1 to Fortitude Saves

- Meme Master: You have studied the art of making memes. You can effortlessly create tier 7 memes that only the most sophisticated and experienced meme experts would understand.
Effect: +1 to Knowledge (World)

2.  Introduction of a subclass that would basically be the “challenge class”. A class for people that wish to challenge themselves more while playing the Isekai Protagonist.

I'd name it the “Mundane Adventurer”:
Description: You were not given any cool powers after being reincarnated into this new world. You are actually pretty normal, all things considered. That does not have to stop you from seeking adventure though, you'll just have to do like your average adventurer and work for it.

I'd imagine the class like this:

- Full BAB
- Spontaneous Caster
- Casting Statt: Charisma
- High Saves
- Hit Die +7
- Alignment: Any
- Gets the same Perk and spell progression as the base Isekai Class
- Gains number of spells known and casts per rest equivalent to the sorcerer class instead of the normal isekai progression
- Can only select underpowered abilities in class progression and only gets 3 of those (Level 1, 10, 20)
- Can only select “Normal Powers” during class progression and only gets 4 of those (Level 5, 10, 15, 20)
- Only Simple weapons proficiency
- Only Light Armour proficiency
- No Pet
- No Plot Armour
- No extra features from the Isekai class (Release Energy, Signature Ability, Beach Episode, Otherwordly Stamia, etc.)
- The exception being fighter training. I'd leave it at level 3.
- Legacy Selection limited to Legacies that only give features that equal the class they are based on. So the Paladin or Monk legacies are OK, since they just copy the class they are based on. The Sorcerer or the Bloodrager legacy, for example, would not be included, since they grant access to too many features, which is inherently very powerful. Too powerful in my opinion, to include it in the “challenge class”. Player Legacie - Part timer is, of course, also a big no-go for this class.
- Capstone: Above Average
Description: You worked hard, despite your less glamorous starting situation in this world, and it shows. You have a distinct edge on your adventuring colleagues.
Effect: +6 to all Statts

Aaand that's about it. Hope I covered everything important and that this is somewhat helpful ^^.

**Comment by CGNefarious** [comment]
There is a huge gap between "Always roll with advantage" and "+1 to a specific skill" on the optimization curve. I'm assuming the people who were suggesting slightly less overpowered options we not asking for necessarily underpowered options, just more options that don't make your character absurdly broken.

**Comment by Necht** [comment]
I think everyone will agree that God-Emperor's ability is much more powerful than the IP.
IP can choose one thing, while God-Emperor gets everything. He is objectively the most powerful archetype and no other can compare with him.

![image](https://github.com/JohN100x1/IsekaiMod/assets/128060492/b69a2573-0c02-4eda-bc61-45940ace4862)
![image](https://github.com/JohN100x1/IsekaiMod/assets/128060492/9f3fdbd6-a98c-45b3-9c3c-32f5ec5aef46)

**Comment by Mythalar** [comment]
@Necht That's simply not true, or at least really not as blatant as that.

An IP can choose many mythic class abilities as special powers, chosen immunities and massive damage boosts (physical or magical). God-Emperor looses all of them.

You also cherry picked the Invicibility hax, which is truly less powerful than the God Emperor Capstone. But the one hit to kill is also a possibility and nothing can compare...

**Comment by JohN100x1** [comment]
> There is a huge gap between "Always roll with advantage" and "+1 to a specific skill" on the optimization curve. I'm assuming the people who were suggesting slightly less overpowered options we not asking for necessarily underpowered options, just more options that don't make your character absurdly broken.

I think so too. Having more options for OP abilities and Special powers would give players to play how they want.
The issue some users take is with the base abilities of the class/subclass, since most of those are not optional. I think it may be time to make a new, less powerful class if the idea of underpowering the current class features remains controversial.

**Comment by Necht** [comment]
> @Necht That's simply not true, or at least really not as blatant as that.
> 
> An IP can choose many mythic class abilities as special powers, chosen immunities and massive damage boosts (physical or magical). God-Emperor looses all of them.
> 
> You also cherry picked the Invicibility hax, which is truly less powerful than the God Emperor Capstone. But the one hit to kill is also a possibility and nothing can compare...

There is no difference in how strong your attacks are, even if you are able to kill everyone with 1 hit, if your opponent has absolute immunity to your attack. It just doesn't make sense. You are helpless against such an enemy because you are unable to harm him. While he, in addition to being immune to everything, also gets a very strong attack buff. And he is capable of injuring and killing you.

Perhaps I don't know something about mechanics, but I sincerely don't understand where I could, I could, according to your words, say "not true, or at least really not as blatant as that".

Or maybe you think that the God-Emperor is not the strongest among them? If this is the case, then I ask you to describe in detail why.

**Comment by Mythalar** [comment]
>There is no difference in how strong your attacks are, even if you are able to kill everyone with 1 hit, if your opponent has absolute immunity to your attack. It just doesn't make sense

Well it's not a duel between IP and God Emperor, so if you kill anything in one hit there is no need to be immune to damage, that was my point.
And even with Invincibility you can, by lvl20, squeeze more damage from IP than from God Emperor with magical amplification, the various "strike" special powers etc... You cannot consider only the capstone.
But God Emperor is very good at support with generalist auras and do not care with enemies immunities (un/holy damage), IP can choose only one of those at lvl 10.

More on the actual point I guess JohN100x1 is right, or at least NightOwlByDay.

**Comment by NightOwlByDay** [comment]
> I think everyone will agree that God-Emperor's ability is much more powerful than the IP. IP can choose one thing, while God-Emperor gets everything. He is objectively the most powerful archetype and no other can compare with him.
> 

Well, I don't agree. In fact, I strongly disagree.

The God-Emporer is, in my opinion, the weakest of the Isekai classes at the moment.
@Mythalar already mostly explained why I think so: It does not have access to Special Powers. The power level of the God-Emperor pales in comparison to what can be achieved through the combination of certain powers from the special power selection.
Sure, the God-Emperor becomes immune to damage at level 20, but you can build an Iseka Protagonist that no enemy can touch with attacks or spells at level 20 as well, which practically has the same effect, **_and_** can clear the entire screen of enemies with one melee attack. Both thanks to Special Powers.
The God-Emperor can't do the second part through the base progression of his class. Unless you have a rather small screen, I suppose.
The gap becomes a lot larger if we go looking at a caster Isekai Protagonist vs a caster God-Emperor.

Furthermore, the feature that makes the God-Emperor invincible comes online only at level 20. Most of the game is not played at level 20, though. So, the performance of the God-Emperor in earlier levels should also be considered.
This only furthers the gap between God-Emperor and the other Isekai classes. Most of which get access to their first Special Power at level 3.
At level 10 the base Isekai class has access to three Special Powers (Edge Lord even 4). That is enough to start realising some very powerful synergies which will put the base Isekai class _far_ ahead of the God-Emperor on the power curve at that level.

In short: The other Isekai classes can imitate the defensive capabilities of the God-Emperor at level 20, while having a sizeable advantage when it comes to offensive capabilities. As such, I consider the God-Emperor the least powerful of the available roster.

Also: I see no point in pitting the classes against each other for determining if one of the features of a given class should be adjusted. Wrath of the Righteous is a single player game. There is no PvP, there are no leaderboards, achievements are blocked if you use this mod. No player is getting an advantage over another player by picking one class or another. The only person affected by that choice is the player themselves. So I see no problem with the classes having differing power levels.

**Comment by Necht** [comment]
@NightOwlByDay @Mythalar 
Well, we look at it differently. I'm not comparing how effective they are in destroying NPCs. All archetypes, obviously, can destroy enemies extremely easily. This is their meaning. And the fact that some archetype does it faster, and some - slower, is absolutely not important within the game.
I compare archetypes with each other and what each of them can and cannot do in comparison with each other. And that's exactly the big catch.

@JohN100x1 
I don't think it's necessary to create a separate weaker class. At least not at this stage. This will do double work for you to further change and improve the mod, because you will have to change 2 architectures at the same time. Perhaps much later, when everything is ready, it will be possible to think about creating a weaker variation of the class. In the meantime, you can just think about a little rebalancing of the current abilities and adding new ones.

If I remember correctly, you also wanted to think about adding a prestige class for the Legend and possibly new companions.

**Comment by Mythalar** [comment]
>All archetypes, obviously, can destroy enemies extremely easily. This is their meaning. And the fact that some archetype does it faster, and some - slower, is absolutely not important within the game.

Well I would argue that it is, in fact, important since you can bypass quite some difficulties by wiping enemies faster, all the more in one hit.
But I guess those comparisons are not really useful so long as there are actually different flavors.

At least the : 
>I think **everyone will agree** that God-Emperor's ability is much more powerful than the IP.

was the actual wrong statement x)

**Comment by Mythalar** [comment]
>Yes, you're right, I didn't think there would be people who would say that this ability would be equal to or weaker than the one that IP has.

Two points : 
1) You now speak of one ability, the capstone, whereas you spoke of the complete archetype before, and my (our, since I guess @NightOwlByDay is more or less on the same line) answer would not have been the same if you had.
2) Yet again you reduce the archetype to the capstone but we are obviously not saying God-Emperor capstone is weaker that IP Hax, just that even if it were true IP is probably stronger as a whole with special powers (load of choices, mythic features from other path etc...), which is not an issue at all by the way since all the class is powerful enough.

But well, to each their own if youd rather no see our point.
I'll let it down there :)

**Comment by Necht** [comment]
@Mythalar Well, I didn't want to continue this, but if you need it so much, then here.

> I think everyone will agree that God-Emperor's ability is much more powerful than the IP. IP can choose one thing, while God-Emperor gets everything. He is objectively the most powerful archetype and no other can compare with him.
> 
> ![image](https://user-images.githubusercontent.com/128060492/237439585-b69a2573-0c02-4eda-bc61-45940ace4862.png) ![image](https://user-images.githubusercontent.com/128060492/237439654-9f3fdbd6-a98c-45b3-9c3c-32f5ec5aef46.png)


I was talking, first of all, about the ability of level 20 and how much it outweighs the ability of the IP.

So no, not "now", but initially.

**Comment by Mythalar** [comment]
>Or maybe you think that the God-Emperor is not the strongest among them? If this is the case, then I ask you to describe in detail why.

My bad then if there you talked about them abilities and not archetypes.
First post duly noted.

**Comment by NightOwlByDay** [comment]
> I think everyone will agree that God-Emperor's ability is much more powerful than the IP. IP can choose one thing, while God-Emperor gets everything. He is objectively the most powerful archetype and no other can compare with him.
> 


"_I think everyone will agree that God-Emperor's ability is much more powerful than the IP. IP can choose one thing, while God-Emperor gets everything._"
That's not the part I disagreed with.

"_He is objectively the most powerful **archetype** and no other can compare with him._"
This is.
This statement was not about the capstones, it was specifically about the archetype.

**Comment by TheIrror** [comment]
I do agree with the Hax conundrum. As it is, I just remove the feature with Toybox when I reach level 20. I am, of course, strictly in favor of the power fantasy flavor that makes the mod so enticing. However, the Hax feature simply renders most of the previous features obsolete, kind of spoiling the journey in my view. When taking into account levels past 20, this issue only worsens. I feel the feature could be refined to still be powerful, but in a way that is interesting to build a character around. I personally only play as the Isekai Protagonist in the unfair difficulty, making me "Overpowered But Overly Cautious" haha. Instakill seems to be in the same vein, and for all the love I have for Supreme Being, it really trivializes any difficulty. I guess those Overpowered Abilities could really benefit from a level gate like Excalibur and Unreactable. Those were very nice; not only do I love the new features, but the level gate on them as well. Truly genius.

Gifted does seem to clash with Meta Luck. One solution could be something that I'm not sure is possible but would involve stacking the rolls, meaning taking the higher of three rolls when you have both abilities. Another would be making Gifted's competence bonus a progressive one like Plot Armor has and leaving only Meta Luck with the rolls. Honestly, all of these abilities are overpowered enough for each to be a feature of their own with no real losses, in actuality, only wins hahaha, as it would lend the mod more build crafting potential. I could see two Meta Lucks, one affecting enemies and the other the player being a real fun split feature, while Gifted can become a scaled feature like Plot Armor, which I agree on halving. I am sure these changes would make you still very overpowered while keeping the game a little challenging and super fun :)

**Comment by NightOwlByDay** [comment]
> Gifted does seem to clash with Meta Luck. One solution could be something that I'm not sure is possible but would involve stacking the rolls, meaning taking the higher of three rolls when you have both abilities.

Good news: That's exactly how it currently behaves ingame :D !

> Instakill seems to be in the same vein, and for all the love I have for Supreme Being, it really trivializes any difficulty. I guess those Overpowered Abilities could really benefit from a level gate like Excalibur and Unreactable.

Supreme Being truly is incredibly powerful, agreed.
The problem I have with levelgating it is, that it only moves the powerspike to a later part of the game.
I'd advocate for making it scale with the  characters level at +1 to all stats per level (+10 as base Stats). That way, we'd still arrive at 30 as baseline at level 20, but wouldn't have the huge power spike in earlier levels.

Sadly, I can't really judge the feasibility of this idea on the technical side of things, since some method would have to constantly  compare the characters current stats to (characters level + 10) and set the attribute in questions to the bigger value of the two. So I don't know if this is a usefull suggestion ...

**Comment by TheIrror** [comment]
That's super cool! I never used Gifted and Meta Luck together because I thought they wouldn't stack. I didn't know it worked like that. Super smooth! English is not my primary language, so maybe I didn't read it well and the text points to them stacking. I will check it out. Otherwise, it would be nice if the feature text hinted at it. Thanks for the tip.

That idea is soooo good! I love it. Making it a progressive growth kind of feature is genius. That type of mechanic is fire. I hope it is feasible; it would make it so much more fun to play. :3
